internal/pubsub/drivertest/drivertest.go

```diff
@@ -193,23 +196,29 @@ func testSendReceive(t *testing.T, newHarness HarnessMaker) {
 		if err := top.Send(ctx, m); err != nil {
 			t.Fatal(err)
 		}
-		want = append(want, m)
+		want = append(want, Message{
+			Body:     string(m.Body),
+			Metadata: m.Metadata,
+		})
 	}
 
 	// Receive from the subscription.
-	var got []*pubsub.Message
+	var got []Message
 	for i := 0; i < len(want); i++ {
 		m, err := sub.Receive(ctx)
 		if err != nil {
 			t.Fatal(err)
 		}
-		got = append(got, m)
+		got = append(got, Message{
+			Body:     string(m.Body),
+			Metadata: m.Metadata,
+		})
 		m.Ack()
 	}
 
 	// Check that the received messages match the sent ones.
-	less := func(x, y *pubsub.Message) bool { return bytes.Compare(x.Body, y.Body) < 0 }
-	if diff := cmp.Diff(got, want, cmpopts.SortSlices(less), cmpopts.IgnoreUnexported(pubsub.Message{})); diff != "" {
+	less := func(x, y Message) bool { return x.Body < y.Body }
+	if diff := cmp.Diff(got, want, cmpopts.SortSlices(less)); diff != "" {
```

Comment:
Why new `.replay` files?

Reply:
I'm not sure how those got into the change. Removing them.